Assets/Plugins/Source/Editor/Windows/CreatePackageWindow.cs

```diff
@@ -301,7 +301,7 @@ private async void StartCreatePackageAsync(UPMUtility.PackageType type)
                 {
                     // Ternary statement here to prevent a divide by zero problem
                     // ever happening, despite how odd it would be in this case.
-                    float newActualProgress = (0.0f >= value.TotalBytesToCopy)
+                    float newActualProgress = (0.0f <= value.TotalBytesToCopy)
```

Comment:
Oh hey, this is familiar! Was this hit in another PR?

Reply:
Yep! Although I duplicated the mistake - so good that you noticed this twice :D

Reply:
Marking this as resolved because updating this PR with `development` put this change out of the PR (and doing so resolved the issue).